packages/utilities/psammead-test-helpers/package.json

```diff
@@ -25,7 +25,12 @@
     "snapshot"
   ],
   "dependencies": {
-    "jest-styled-components": "^6.3.1",
-    "react-test-renderer": "^16.6.3"
+    "jest-styled-components": "^6.3.1"
+  },
+  "devDependencies": {
+    "react": "^16.6.3",
+    "react-dom": "^16.6.3",
+    "react-test-renderer": "^16.6.3",
+    "react-testing-library": "^5.3.1"
```

Comment:
As this is a dev library, I think these should be `dependencies` not `devDependencies`